### PR TITLE
Activate "submit building hour update" button

### PR DESCRIPTION
**Pull request by hawkrives**
So submitting a GitHub issue, while nice, isn't really workable – we'd either need to make our users set up and log in to a github account, or ship our app's private token with the app somehow, which would allow someone to extract it from the app and pretend to be the app (not the end of the world, but not ideal).

So I went with email. `allaboutolaf@stolaf.edu` currently emails @drewvolz, me, and @rye; lmk if you want to be added (or removed).

Closes https://github.com/StoDevX/AAO-React-Native/issues/1189.

### CODE REVIEW

**Comment by hawkrives** [comment]
I stole like a quarter of this code from https://github.com/StoDevX/AAO-React-Native/issues/1189.

**Comment by rye** [comment]
@hawkrives I'd be down to be added.  Always nice to keep my ear to the ground, if nothing else.  It seems I've already been added.

From my own experience, it seems there is no really easy way to ship a private GitHub API key to the world as suggested besides doing just that; from GitHub's perspective I'm sure having many devices using the token for direct API access without any rate limiting or filtering in the middle is not ideal.  Hence, either you write a little API yourself to handle it (take JSON payloads from clients, process and send to GitHub) or you just use email—the latter is far easier.

**Comment by hawkrives** [comment]
I want to add a link to the email that creates a github issue for us in one click: https://eric.blog/2016/01/08/prefilling-github-issues/

For example, https://github.com/StoDevX/AAO-React-Native/issues/new?&labels=data/hours&title=[building]%20Suggestion%20for%20&body=This%20is%20a%20prefilled%20issue

That way we can still easily collect reports into our issue tracker.

**Comment by drewvolz** [comment]
Including a prefilled link for quickly creating the issue sounds like a great idea. Thanks for finding that.

**Comment by elijahverdoorn** [comment]
I'd like to be added to the alias.

**Review by elijahverdoorn — APPROVED** [review]

File: source/views/building-hours/report/submit.js [hunk 1]
```diff
@@ -20,22 +20,44 @@ export function submitReport(current: BuildingType, suggestion: BuildingType) {
   )
 }
 
-function makeIssueBody(before: string, after: string): string {
+
+function makeEmailBody(before: string, after: string, title: string): string {
   return dedent`
+    Hi! Thanks for letting us know about an hour change.
```
Comment (drewvolz) [inline source/views/building-hours/report/submit.js]: Maybe substitute "hour" with "schedule"? Because they could change more than just hours.

Comment (drewvolz) [reply source/views/building-hours/report/submit.js]: Make this relevant as it is now shown as outdated...

**Pull request merged**